src/meili/routing.cc

```diff
@@ -572,18 +574,21 @@ find_shortest_path(baldr::GraphReader& reader,
           if (endtile == nullptr) {
             continue;
```

Comment:
consider putting endtile (and check for null) inside conditional for adding label. The vast majority case is that the label will not be added due to exceeding cost or time threshold, and very small chance of endtile being null (and saving computation of cost).